docs/user-guide/deploy-on-bare-metal.md

```diff
@@ -38,91 +38,21 @@ _*Note*: This method could be used also in Virtual environment_
 
 ## iPXE
 
-iPXE deployment method
+*NOTE*: We use a sample URL, please change to fit your use case accordingly
 
-*NOTE1*: We use a sample URL, please change to fit your use case accordingly
-*NOTE2*: We've set the live_url as the node hostname on 8080 port , please change to fit your use case accordingly
+### Accessing the iPXE boot script and artifacts
 
-### Automatic
+The service serves an iPXE boot script for each infra-env
```

Comment:
It might be nice to link to more info about iPXE - maybe the [enhancement](https://github.com/openshift/assisted-service/blob/master/docs/enhancements/ipxe-host-boot.md) doc?

Reply:
I don't know if we want to link to enhancements from user docs. They have a lot of extra information that a user really wouldn't need and would have to sift through.

What exactly do you think we should add here? Maybe what endpoint the script is served on?

Reply:
That makes sense that it would be too much information. 

Yes, maybe the format of the endpoint like the one in the enhancement `GET /v2/infra-envs/{infra-env-id}/downloads/files?file_name=ipxe-script` so that way the user knows which parts of the example URL they need to modify